.travis.yml

```diff
@@ -13,7 +13,7 @@ rust:
   - nightly
   - beta
   - stable
-  - 1.38.0
+  - 1.43.0
```

Comment:
The idea is that sometimes incompatibilities are introduced in newer versions and this might catch that. However, that requires a CI cron job, which hasn't been set up so may as well remove them and speed up the build.